reps/2022-11-23-labels-mechanish-and-affinity-schedule-feature.md

```diff
@@ -0,0 +1,382 @@
+## Summary
+### General Motivation
+
+Introduce the Labels mechanism. Give Labels to Actors/Tasks/Nodes/Objects. 
```

Comment:
Introduce labels mechanism, which associate an enumerated property to Ray nodes. We can static assign labels to Ray node on start up, or dynamically change node's label through Ray scheduling APIs. 

Labels mechanism makes it easy to implement Actor and Node affinity.([Taints and Tolerations style](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/))